scripts/deploy-upgradable.js

```diff
@@ -2,18 +2,24 @@
 require('dotenv').config();
 const _ = require('lodash/fp');
 const {
+    Contract,
     Wallet,
     getDefaultProvider,
     utils: { isAddress },
 } = require('ethers');
-const { deployUpgradable, upgradeUpgradable, getProxy, predictProxyAddress } = require('./upgradable');
 const readlineSync = require('readline-sync');
 const { outputJsonSync } = require('fs-extra');
+const { deployUpgradable, upgradeUpgradable } = require('@axelar-network/axelar-gmp-sdk-solidity');
```

Comment:
The `upgradable.js` in gmp sdk repo should be updated to with the changes present in this one before we delete it. For e.g. `predictProxyAddress` is being removed here. Can you do a diff and port over the changes there?

Reply:
They those methods are already present in `gmp-sdk`

Reply:
Fixed by using `predictContractConstant` directly from `SDK`